tests/blueprint_with_behavior.ts

```diff
@@ -0,0 +1,92 @@
+export const bpBuilding1 = blueprint.building2x1_2M_1({
+    name: "Bot Factory",
+    medium: [
+        component.roboticsAssembler([
+            {name: "craft1", to: "active1"},
+            {to: visual},
```

Comment:
How would you link from `visual` to a component's register?
Would that just be by specifying `visual: {to: "foo"}` at the top level of the behavior?

Reply:
Correct, together with `{name: "foo"}` for the targeted register.